roxctl/deployment/check/check.go

```diff
@@ -123,12 +125,18 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	utils.Must(c.Flags().MarkDeprecated("print-all-violations", "use the new output format where all "+
 		"violations are printed by default. This flag will only be relevant in combination with the --json flag"))
 
+	// We need a string parameter with the filenames for the sarif printer factory
+	// If a report is based on a single yaml file it will work as previously, if multiple
+	// they will be concatenated as csv
+	deploymentCheckCmd.joinedFiles = strings.Join(deploymentCheckCmd.files[:], ",")
```

Comment:
Why is the slice notation used here?
```suggestion
	deploymentCheckCmd.joinedFiles = strings.Join(deploymentCheckCmd.files, ",")
```

Reply:
I thought it was necessary, but now that I tested it leaving it out the slice notation leads to the same result. I've removed it now.